infra/logging/src/main/java/com/simprints/infra/logging/Simber.kt

```diff
@@ -176,4 +108,105 @@ object Simber {
         it is SSLProtocolException ||
         it is SSLHandshakeException ||
         it is FirebaseNetworkException
+
+    companion object {
+        const val DEFAULT_TAG = "Simber"
+        const val USER_PROPERTY_TAG = "zzUserPropertyTag"
+        private const val FIREBASE_ANALYTICS_MAX_MESSAGE_LENGTH = 100
+        private const val FIREBASE_ANALYTICS_MAX_TAG_LENGTH = 24
+        private const val FIREBASE_ANALYTICS_MAX_USER_TAG_LENGTH = 40
+        private val firebaseInvalidCharactersRegex = Regex("[^a-zA-Z0-9_]")
+
+        val INSTANCE = Simber()
```

Comment:
Not that the current setup is ideal but this feels even more awkward

Reply:
Only because the initial Simber interface (specifically the tag application) was modelled after Timber and I did not want another project-wide refactoring for logs in this PR. 

If we move the tags to be parameters in Simber methods, all those companion hacks could be removed. I can do it in a separate PR, after this is merged.